src/files.js

```diff
@@ -73,7 +73,7 @@ const NewFilePlugin = {
 		const isPublic = document.getElementById('isPublic') ? document.getElementById('isPublic').value === '1' : false
 		if (isPublic) {
 			return window.FileList.createFile(filename).then(function() {
-				OCA.Viewer.open(path)
+				OCA.Viewer.open(path.replace(/&/g, '%26'))
```

Comment:
I could not reproduce any issue with that and the `OCA.Viewer.open` call. Maybe @skjnldsv knows if there was a recent fix in viewer itself?

Imo this should be handled by the viewer app anyways.

Reply:
Ok, the issue is actually not in the viewer open but in the component we register when we generate the URLs for opening the document. See https://github.com/nextcloud/officeonline/pull/69 for a fix